exchanges/gateio/gateio_ws_delivery_futures.go

```diff
@@ -104,7 +103,7 @@ func (g *Gateio) GenerateDeliveryFuturesDefaultSubscriptions() ([]stream.Channel
 			futuresBalancesChannel,
 		)
 	}
-	pairs, err := g.GetEnabledPairs(asset.DeliveryFutures)
+	pairs, err := g.FetchTradablePairs(context.Background(), asset.DeliveryFutures)
```

Comment:
Instruments in the config files are not perpetual(they expire) and the test is using these instruments when running the tests

Reply:
I understand but we should be calling FetchTradablePairs in the test file and making sure we are updating our enabled list, instead of adding a friction point when these should already be updated on startup.